.travis.yml

```diff
@@ -29,7 +29,11 @@ cache:
   - pip
 
 python:
-  - "3.5"
+  - "2.7"
+  - "3.6"
+
+matrix:
+  fast_finish: true
```

Comment:
Please change to:

```
language: python

cache:
  - pip

matrix:
  fast_finish: true
  include:
    - python: 2.7
    - python: 3.6
    - python: 3.7
      dist: xenial
```

as it will be good to get Python 3.7 badge already :wink: